src/google-classroom-extractor/google_classroom_extractor/student_submissions.ipynb

```diff
@@ -0,0 +1,255 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 140,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib\n",
+    "matplotlib.style.use('ggplot')"
```

Comment:
Line 11 threw an error on my system, which I was able to resolve using `%matplotlib inline` as suggested by https://stackoverflow.com/questions/27204646/matplotlib-pyplot-has-no-attribute-style